Apps/Immunization/src/MapUtils/ImmunizationRecommendationMapUtils.cs

```diff
@@ -75,10 +75,10 @@ private static ImmunizationRecommendation FromPhsaModel(string recommendationSet
             {
                 RecommendedVaccinations = model.TargetDisease == null ? recommendedVaccinations : string.Empty,
                 RecommendationSetId = recommendationSetId,
-                DiseaseEligibleDate = diseaseEligible?.Value != null ? DateTime.Parse(diseaseEligible.Value, CultureInfo.CurrentCulture) : null,
-                DiseaseDueDate = diseaseDue?.Value != null ? DateTime.Parse(diseaseDue.Value, CultureInfo.CurrentCulture) : null,
-                AgentEligibleDate = agentEligible?.Value != null ? DateTime.Parse(agentEligible.Value, CultureInfo.CurrentCulture) : null,
-                AgentDueDate = agentDue?.Value != null ? DateTime.Parse(agentDue.Value, CultureInfo.CurrentCulture) : null,
+                DiseaseEligibleDate = diseaseEligible?.Value != null ? DateOnly.Parse(diseaseEligible.Value, CultureInfo.CurrentCulture) : null,
```

Comment:
So how is the date coming over frmo PHSA?  Just zeros for the time portion?

Reply:
These 4 are coming like this:

```
"dateCriterion": [
  {
    "dateCriterionCode": {
      "text": "Forecast by Disease Due Date"
    },
    "value": "2011-08-20"
  },
  {
    "dateCriterionCode": {
      "text": "Forecast by Disease Eligible Date"
    },
    "value": "2011-08-20"
  }
]
```